x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/predicate/nulls/IsNotNull.java

```diff
@@ -37,7 +39,42 @@ public class IsNotNull extends UnaryScalarFunction implements EvaluatorMapper, N
         IsNotNull::new
     );
 
-    public IsNotNull(Source source, Expression field) {
+    @FunctionInfo(
+        description = "Is *not* this null?",
```

Comment:
```suggestion
        description = "Filters for documents which contain the specified field.",
```

nit: descriptions should start with verbs

Reply:
I guess the tricky bit is with assigned/computed values, which aren't necessarily fields in a doc (but derived from them).

Reply:
Verb phrases are so long here! But sure.

Reply:
Wait. It doesn't *keep* anything. It makes `true` or `false`. It answers a yes/no question. It only keeps things in a WHERE clause - but folks will use it in `EVAL` for stuff like `CASE(a IS NOT NULL, a, b)`. Yes, that one is easier to write as COALESCE, but still, you can use them both places.

Reply:
yeah thanks for clarifying, as always just suggestions from me I'll always defer to SME, but I think it makes sense that descriptions are formulated consistently 

I like where you landed here TBH  👍